### PR TITLE
feat: added method for getting ordered quantity in the supplier score…

### PR DESCRIPTION
**Pull request by Vishnu7025**
feat: added method for getting ordered quantity in the supplier scorecard variable.
@ankush

no-docs

### CODE REVIEW

File: erpnext/buying/doctype/supplier_scorecard_variable/supplier_scorecard_variable.py [hunk 1]
```diff
@@ -422,6 +424,29 @@ def get_total_shipments(scorecard):
...
+			(po.transaction_date >= scorecard.get("start_date")) & (po.transaction_date <= scorecard.get("end_date"))
+		)
+		.run(as_list=1)
+	)[0][0]
+
+	if not data:
+		data = 0
+	return data
+
+
```
Comment (s-aga-r) [inline erpnext/buying/doctype/supplier_scorecard_variable/supplier_scorecard_variable.py]:
```suggestion
def get_ordered_qty(scorecard):
    """Returns the total number of ordered quantity (based on Purchase Orders)"""

    po = frappe.qb.DocType("Purchase Order")

    return (
        frappe.qb.from_(po)
        .select(Sum(po.total_qty))
        .where(
            (po.supplier == scorecard.supplier)
            & (po.docstatus == 1)
            & (po.transaction_date >= scorecard.get("start_date"))
            & (po.transaction_date <= scorecard.get("end_date"))
        )
    ).run(as_list=True)[0][0] or 0


```

**Review by s-aga-r — CHANGES_REQUESTED** [review]

**Comment by Vishnu7025** [comment]
@s-aga-r

**Comment by s-aga-r** [comment]
> @s-aga-r 

@Vishnu7025 also fix checks

**Comment by deepeshgarg007** [comment]
@Vishnu7025 Please setup https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist#linting locally and resolve linting issues. Reopen once done

**Pull request closed without merging**